src/Symfony/Component/HttpFoundation/Exception/InvalidTrustedHeaderException.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpFoundation\Exception;
+
+final class InvalidTrustedHeaderException extends \InvalidArgumentException implements ExceptionInterface
+{
+    /**
+     * @var string
+     */
+    private $header;
+
+    /**
+     * @var string
+     */
+    private $value;
```

Comment:
thank, indeed, however as opt [here](https://github.com/symfony/symfony/pull/20389#discussion_r86617704) I will remove the class all together.